docs/excel/custom-functions-debugging.md

```diff
@@ -41,22 +44,22 @@ You can use VS Code to debug UI-less custom functions in Office Excel on the des
 ### Run your add-in from VS Code
 
 1. Open your custom functions root project folder in [VS Code](https://code.visualstudio.com/).
-2. Choose **Terminal > Run Task** and type or select **Watch**. This will monitor and rebuild for any file changes.
-3. Choose **Terminal > Run Task** and type or select **Dev Server**.
+1. Choose **Terminal > Run Task** and type or select **Watch**. This will monitor and rebuild for any file changes.
+1. Choose **Terminal > Run Task** and type or select **Dev Server**.
```

Comment:
I think you only need to bold the actual term (e.g., see [guidance for Tabs](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=29014))
```suggestion
1. Choose **Terminal** > **Run Task** and type or select **Watch**. This will monitor and rebuild for any file changes.
1. Choose **Terminal** > **Run Task** and type or select **Dev Server**.
```